test/extended/testdata/imagestreamtag-jenkins-slave-pods.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: jenkins-slave
+spec:
+  tags:
+  - from:
+      kind: DockerImage
+      name: docker.io/openshift/jenkins-slave-maven-centos7:latest
+    name: base
+  - annotations:
+      role: jenkins-slave
+      slave-label: jenkins-slave
```

Comment:
left the label the same to keep the app consistent but changed the imagestream name